src/OutputFormat/CalculationBehaviour.jl

```diff
@@ -517,29 +517,36 @@ where,
 
 """
 function getSurchargePressure(behaviour, P::Array{Float64}, PP::Array{Float64})
-    dxx = 0.0
-    wt = PP[behaviour.bottomPointIndex]
-    pressure1 = behaviour.appliedPressure - wt
-    pressure = pressure1
+    # Depth
+    Δx = 0.0
+    # Effective stress at bottom of foundation
+    σ_bottom = PP[behaviour.bottomPointIndex]
+    # Net applied footing pressure
+    Qnet = behaviour.appliedPressure - σ_bottom
+
+    # Loop through each nodal point below foundation
     for i=behaviour.bottomPointIndex:behaviour.nodalPoints
-        if dxx < 0.01
-            P[i] += pressure
-            dxx += behaviour.dx
+        if Δx < 0.01 # TODO: Why was this hardcoded to 0.01? Make this a constant. Maybe named MIN_DX?
```

Comment:
Good question @EmilSoleymani.  I'm really glad you are looking at the code this closely!